app/services/spree_shopify_importer/data_savers/tax_categories/saver.rb

```diff
@@ -0,0 +1,19 @@
+module SpreeShopifyImporter
+  module DataSavers
+    module TaxCategories
+      class Saver
```

Comment:
what about keeping convention and tax_category_creator.rb?

Reply:
oh, sure, I'll change it to creator. But do you think that we need to repeat the name in the module and in the class name?
I mean module `TaxCategories` and class `TaxCategoryCreator` vs module `TaxCategories`  and class `Creator`?
I think that we created a module in order to understand the context and we don't need repetition.

Reply:
Oh I would go with something like this 
SpreeShopifyImporter::DataSavers::TaxCategory::Create
SpreeShopifyImporter::DataSavers::TaxCategory::Update
but again if other services are called SpreeShopifyImporter::DataSavers::Itemss::ItemCreator we should stick with it, and eventually do a global refactor.

Reply:
So yes it doesn't make much sense to repeat it, but this is some kind of convention in this code and we should stick with it, or change convention to the new one - but not mix them under one namespace.

Reply:
@kgorazd  can I create a separate PR and change the convention, for example today?
I don't like this repetition.

Reply:
so, I'll change the name to `tax_category_creator.rb`but may be later, let's discuss how to change it.